package.json

```diff
@@ -0,0 +1,627 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server",
```

Comment:
[Optional] I suggest that you specify a script (e.g start or dev) to update the render when the files change. e.g 

```javascript
"start": "webpack serve --open"
```

This will help such that running `npm run start` launches the page and updates when your files change and you wouldn't necessarily have to rebuild.